R/az_hourly.R

```diff
@@ -91,7 +91,8 @@ az_hourly <- function(station_id = NULL, start_date_time = NULL, end_date_time =
       as.numeric
     )) %>%
     dplyr::filter(.data$meta_station_id != "az99") %>%
-    dplyr::mutate(date_datetime = lubridate::ymd_hms(.data$date_datetime)) %>%
+    dplyr::mutate(date_datetime = lubridate::force_tz(lubridate::ymd_hms(.data$date_datetime), tzone = "America/Phoenix")) %>%
+    dplyr::mutate(wind_2min_timestamp = lubridate::with_tz(lubridate::parse_date_time(.data$wind_2min_timestamp, orders = "ymdHMSz"), tzone = "America/Phoenix")) %>%
```

Comment:
What's the difference between `force_tz()` and `with_tz()`?

Reply:
NEWS.md file updated

Reply:
Can you commit and push the NEWS.md changes?  I don't see them in this pull request

Reply:
Ah, they're there now. But, I managed to make a conflict

Reply:
I can show you how to deal with the merge conflict next time we meet (next Friday), or you can pick a time tomorrow or earlier next week: https://calendar.app.google/eCWobQteVLViq5NJ8
It'll be an quick fix, just something easier to show over Zoom than via email.